src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/VariantFmsDTOValidator.java

```diff
@@ -176,7 +167,29 @@ public Long validateVariant(VariantFmsDTO dto, List<Long> idsAdded, BackendBulkD
 			}
 			variant.getCrossReferences().addAll(xrefs);
 		}
-		
+
+		if (CollectionUtils.isNotEmpty(dto.getReferences())) {
+			Set<Reference> validatedReferences = new HashSet<>();
+			for (PublicationRefFmsDTO refDto : dto.getReferences()) {
+				if (ObjectUtils.isNotEmpty(refDto.getPublicationId())) {
+					Reference reference = referenceService.retrieveFromDbOrLiteratureService(refDto.getPublicationId());
+					if (reference == null) {
+						variantResponse.addErrorMessage("references - publicationId", ValidationConstants.INVALID_MESSAGE + " (" + refDto.getPublicationId() + ")");
+					} else {
+						validatedReferences.add(reference);
+					}
+				}
+			}
+			if (CollectionUtils.isNotEmpty(validatedReferences)) {
+				if (variant.getReferences() == null) {
+					variant.setReferences(new ArrayList<>());
+				} else {
+					variant.getReferences().clear();
+				}
+				variant.getReferences().addAll(validatedReferences);
+			}
+		}
```

Comment:
What happens if `dto.getReferences()` is empty?  You're not clearing any existing references from the database.  There is code in the AlleleDTOValidator that can basically be copied and pasted here.

Reply:
Changed as AlleleDTOValidator